doc/user/options.md

```diff
@@ -125,7 +125,17 @@ Native switches:
 ## Determining the TruffleRuby home
 
 TruffleRuby needs to know where to locate files such as the standard library.
-These are stored in the TruffleRuby home directory. The TruffleRuby option
-`home` has priority for setting the home directory. Otherwise it is set
-automatically to the directory containing the TruffleRuby JAR file, if
-TruffleRuby is running on a JVM.
+These are stored in the TruffleRuby home directory.
+
+The search priority for finding Ruby home is:
+
+* The value of the TruffleRuby `home` option.
```

Comment:
I think clarifying what you mean by "option" here would be helpful:
```
The value of the TruffleRuby `home` option (i.e., `-Xhome=path/to/home`).
```

Reply:
Fixing.